src/wallet/test/crypto_tests.cpp

```diff
@@ -80,8 +81,8 @@ bool OldDecrypt(const std::vector<unsigned char>& vchCiphertext, CKeyingMaterial
     if (fOk) fOk = EVP_DecryptUpdate(ctx, &vchPlaintext[0], &nPLen, &vchCiphertext[0], nLen) != 0;
     if (fOk) fOk = EVP_DecryptFinal_ex(ctx, (&vchPlaintext[0]) + nPLen, &nFLen) != 0;
     EVP_CIPHER_CTX_cleanup(ctx);
-
     EVP_CIPHER_CTX_free(ctx);
+    ERR_remove_thread_state(nullptr);
```

Comment:
I'm not sure I understand this. OpenSSL API is confusing, where is the symmetric "create" for this remove/free?
Surprised that this needs `ERR_*` just to deallocate a structure that comes out of the blue.
Is this related to cleanup in `~CInit()` in util.cpp?